src/jit/codegencommon.cpp

```diff
@@ -8061,6 +8173,14 @@ void CodeGen::genFinalizeFrame()
         regSet.rsSetRegsModified(RBM_INT_CALLEE_SAVED & ~RBM_FPBASE);
     }
 
+#ifdef UNIX_AMD64_ABI
+    // On Unix x64 we also save R14 and R15 for ELT profiler hook generation.
+    if (compiler->compIsProfilerHookNeeded())
+    {
+        regSet.rsSetRegsModified(RBM_R14 | RBM_R15);
```

Comment:
>       regSet.rsSetRegsModified(RBM_R14 | RBM_R15) [](start = 1, length = 50)

Rather than referring args regs directly, i would suggest to add #ifdef constants in target.h and use preprocessor constant names

e.g.
#define REG_PROFILER_ENTER_ARG0   REG_R14
#define RBM_PROFILER_ENTER_ARG0   RBM_R14

This would allows us to change the register used for these args in one place.